src/components/Home/index.test.tsx

```diff
@@ -0,0 +1,42 @@
+import { render, screen } from '@testing-library/react';
+
+import { postListTestIds } from 'components/Post/List';
+import { getAllPosts } from 'lib/post';
+
+import Home, { homeDataTestIds } from '.';
+
+describe('Home', () => {
+  it('renders the heading', () => {
+    const posts = getAllPosts();
+
+    render(<Home posts={posts} currentPage={1} totalPages={1} />);
+
+    const heading = screen.getByTestId(homeDataTestIds.heading);
+
+    expect(heading).toBeVisible();
+  });
+
+  describe('PostList', () => {
+    describe('when there are no posts', () => {
```

Comment:
```suggestion
    describe('when there are NO posts', () => {
```

Reply:
Changed in https://github.com/nimblehq/til-web/pull/49/commits/4fe4b1ea4478e287b5a0617260fcce38e6d0837f